helm/examples/azure/hpcc-azurefile/values.yaml

```diff
@@ -3,6 +3,7 @@ common:
   mountPrefix: "/var/lib/HPCCSystems"
   #secretName: "azure-secret"
   #secretNamespace: "default"
+  provisioner: "file.csi.azure.com" # Set to "kubernetes.io/azure-file" if  not use csi driver but it is not recommaned.
```

Comment:
trivial: double space between 'if' and 'use'

Reply:
>if  not use csi driver

if not using the csi driver

Reply:
typo: recommaned -> recommended

Reply:
fixed